src/jit/emitxarch.cpp

```diff
@@ -10793,6 +10863,11 @@ size_t emitter::emitOutputInstr(insGroup* ig, instrDesc* id, BYTE** dp)
             assert(code & 0x00FF0000);
 
 #ifdef FEATURE_AVX_SUPPORT
+            if (TakesRexWPrefix(ins, size))
+            {
+                code = AddRexWPrefix(ins, code);
+            }
+
             if (TakesVexPrefix(ins))
```

Comment:
Why was this change necessary? It doesn't look like you added support for any instructions that take a REX prefix.